internal/core/data/interfaces/mocks/DBClient.go

```diff
@@ -78,20 +101,6 @@ func (_m *DBClient) CloseSession() {
 	_m.Called()
 }
 
-// Connect provides a mock function with given fields:
```

Comment:
Is this removal of Connect() method intended to be part of CBOR event handling PR?

Reply:
This was done by the mockery tool. Since I updated the `DBClient` interface I had to use mockery to regenerate the `mock/DBClient`. The `Connect` function was removed from the `DBClient` in [this commit](https://github.com/edgexfoundry/edgex-go/commit/963a78a17580fc1390acdd718979617b762ec255), which causes the code generation tool to not include it. We can add it back if it is needed.